### DISCUSSION TITLE
Clean the Cruft

### DISCUSSION THREAD

**Discussion by cyanlemons** [status-im/status-legacy#16620] [General]
The status-react repository has a lot of cruft. It takes roughly two pages worth of scrolling to even reach the beginning of the README. As part of a renewed effort to make our GitHub account & repos more palatable to newcomers and developers alike, we should make sure we don't leave unnecessary files laying around in our repo, particularly in the root folder.

After a quick observation, I've identified the following:

- The [VERSION](https://github.com/status-im/status-react/blob/develop/VERSION) file is out of date with the contents being `1.8.0`. However, everything seems to be working just fine in the v1.9 release – which begs the question: do we even need this file? Can we remove it?
- There are six different `.env` files in the root folder of the repository. Can we move this to a shared folder? Or perhaps move five of them to a shared folder while keeping the primary `.env` in the root folder?
- The following files haven't been changed in 2-3 years: `supervisord.conf`, `.dockerignore`, `.buckconfig`, `.babelrc`, `.mailmap`, `.nycrc`, `.watchmanconfig`, `app.json`. Do we still need all of these? If not, let's remove the ones we can.
- Should we even be storing a `.vscode` directory in our repo? This seems like a personal preference thing that varies from developer to developer. IMO, this type of folder should just go in the `.gitignore` and then anyone can add all preferences they like without committing it upstream.

The cruft is one problem among many:
- Terribly outdated README
- Broken testing/coverage badges in the README
- Outdated documentation (after a quick look around I found some references to macOS *catalina* workarounds)
- Outdated GitHub profile picture/description
- Swarms are dead, so we should unpin that repository from the `status-im` account

Ideally, we should get around to resolving all of these to give a better impression to our users & SNT holders. Cleaning the cruft felt like a good place to start.

**Comment by cammellos** [comment]
Partially addressed, I will move the rest to a discussion
